community/s6-linux-init/post-install

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh -e
+
+if [ ! -e /etc/s6-linux-init/current ]; then
+    s6-linux-init-maker /etc/s6-linux-init/current
```

Comment:
Can this not be created /in/ the `build` file so that the package manager tracks it via its etcsums handling?

Reply:
let me try it

Reply:
Right, I remember doing that, and it generates fifo files which kiss doesn't handle well (in this case, the fifos are skipped by kiss. In my other test packages kiss would hang if there are fifo files).
I can try make a pr for kiss to make it handle fifos, and this package can wait until that's fixed?

Reply:
I'll look into this. :+1:

Reply:
Question; what exactly is wrong with the handling of fifos?

Reply:
I see. The package manager hangs during installation.

Reply:
Ah. We cannot copy fifos. It's fine to create this during `post-install`. :+1: